nix/cardano-services/deployments/default.nix

```diff
@@ -200,6 +201,8 @@ in
             ];
           };
 
+          backend.allowedOrigins = allowedOrigins ++ ["localhost"];
```

Comment:
I think a string (`allowedOrigins`) is added to a list `["localhost"]`, and `nix run .#⟨…⟩.plan` [is failing](https://github.com/input-output-hk/cardano-js-sdk/actions/runs/8477293822/job/23228036808?pr=1187#step:7:274):

```
       error: value is a string while a list was expected
```